scripts/install_all.sh

```diff
@@ -0,0 +1,193 @@
+        *)
+            echo "$(uname) is an unsupported operating system." >&2
+            echo "You can try to provide a package manager command with \$INSTALL_CMD," >&2
+            echo "but your mileage may vary." >&2
+            exit 1
+            ;;
+    esac
+fi
+
+if ! command -v docker > /dev/null; then
+    echo "Installing Docker..."
+
+    # On macOS, install docker desktop with Homebrew cask
+    if [ $BREW ]; then
+        $INSTALL_CASK_CMD docker > $OUTPUT
+    else
+        $INSTALL_CMD docker.io > $OUTPUT || (echo "You must manually install docker" && exit 1)
+
+        echo "Fixing docker permissions..."
+        sudo groupadd -f docker
+        sudo usermod -aG docker "$USER"
+    fi
+fi
+
+if ! command -v pip3 > /dev/null && ! [ $BREW ]; then # Don't reinstall python on macOS
+    echo "Installing python and pip..."
+    $INSTALL_CMD python3-pip > $OUTPUT || (echo "You must manually install python and pip" && exit 1)
+fi
+
+if ! command -v terraform > /dev/null; then
+    echo "Installing terraform..."
+    if [ $BREW ]; then
+        $INSTALL_CMD terraform > $OUTPUT
+    elif [ $APT ] || confirm "Would you like to automatically install Terraform for amd64 linux?"; then
+        $INSTALL_CMD unzip > $OUTPUT
+        curl -0s "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip" \
+             > "terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
+        sudo unzip -d /usr/bin "terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
+        sudo chmod a+rx /usr/bin/terraform
+	rm "terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
+    else
+        echo "You need to manually install Terraform before continuing..." >&2
+        exit 1
+    fi
+fi
+
+if ! command -v nomad > /dev/null; then
+    echo "Installing nomad..."
+    if [ $BREW ]; then
+        $INSTALL_CMD nomad > $OUTPUT
+    else
+        sudo ./install_nomad.sh
+    fi
+fi
+
+if ! command -v git-crypt > /dev/null; then
+    echo "Installing git-crypt..."
+    $INSTALL_CMD git-crypt > $OUTPUT || (echo "You must manually install git-crypt" && exit 1)
+fi
+if ! command -v jq > /dev/null; then
+    echo "Installing jq..."
+    $INSTALL_CMD jq > $OUTPUT || (echo "You must manually install jq" && exit 1)
+fi
+
+if ! command -v ip > /dev/null; then
+    if [ $BREW ]; then
+        $INSTALL_CMD iproute2mac > $OUTPUT
+    else
+        $INSTALL_CMD iproute2 > $OUTPUT || (echo "You must manually install iproute2" && exit 1)
+    fi
+fi
+
+echo "Creating virtual environment..."
+./create_virtualenv.sh > $OUTPUT
+echo "Run \`source dr_env/bin/activate\` to activate the virtual environment."
+
+echo "Starting postgres and installing the database..."
+./run_postgres.sh > $OUTPUT
+./install_db_docker.sh > $OUTPUT
+
+echo "Starting elasticsearch and building the ES Indexes..."
+./run_es.sh > $OUTPUT
+./run_manage.sh search_index --rebuild -f > $OUTPUT
+
+echo "Updating common dependencies..."
+./update_models.sh > $OUTPUT
```

Comment:
Doesn't this script require the virtualenv to be active?

Reply:
I think these both run inside docker, so they don't need the virtualenv.